_topic_map.yml

```diff
@@ -1241,8 +1241,15 @@ Topics:
     File: creating-machineset-gcp
   - Name: Creating a machine set on OpenStack
     File: creating-machineset-osp
+  - Name: Creating a machine set on RHV
+    File: creating-machineset-rhv
+    Distros: openshift-enterprise,openshift-webscale
+  - Name: Creating a machine set on oVirt
```

Comment:
This is adding in a whole new oVirt assembly that wasn't being added in this PR before (I don't think). Perhaps this was accidentally included when you resolved conflicts?

Reply:
please note the different distros. In OKD we want to reference the upstream project oVirt while in OCP we want to reference the downstream product RHV.

Reply:
> This is adding in a whole new oVirt assembly that wasn't being added in this PR before (I don't think). Perhaps this was accidentally included when you resolved conflicts?

@bergerhoffer My intention is just to have the one assembly, but in the OKD, we want the TOC to say "oVIrt" instead of "RHV". Did I do this incorrectly?
I'm not sure what you mean by a whole new oVirt assembly. I think it's the sa

Reply:
Thanks for the explanations, I didn't realize it was the same assembly, just with a different title depending on the distro.